moveit_core/collision_detection_fcl/src/collision_common.cpp

```diff
@@ -160,6 +160,21 @@ bool collisionCallback(fcl::CollisionObjectd* o1, fcl::CollisionObjectd* o2, voi
   {
     if (cd1->ptr.ab->getAttachedLink() == cd2->ptr.ab->getAttachedLink())
       always_allow_collision = true;
+    else
+    {
+      const std::set<std::string>& tl1 = cd1->ptr.ab->getTouchLinks();
+      const std::set<std::string>& tl2 = cd2->ptr.ab->getTouchLinks();
+      if (tl1.find(cd2->getID()) != tl1.end() || tl2.find(cd1->getID()) != tl2.end())
```

Comment:
You can now use `tl1.contains(cd2->getID())` and vice versa

Reply:
Ah, bummer. I think it may require adding c++20 support in the cmakelists.txt. Never mind then.